encode.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	vegeta "github.com/tsenart/vegeta/lib"
+)
+
+const (
+	encodingCSV  = "csv"
+	encodingGob  = "gob"
+	encodingJSON = "json"
+)
+
+type decoderFunc func(io.Reader) vegeta.Decoder
+type encoderFunc func(io.Writer) vegeta.Encoder
+
+func encodeCmd() command {
+	var (
+		fs     = flag.NewFlagSet("encode", flag.ExitOnError)
+		from   = fs.String("from", "gob", "Input decoding [csv, gob, json]")
+		to     = fs.String("to", "json", "Output encoding [csv, gob, json]")
+		output = fs.String("output", "", "Output file")
```

Comment:
The default output should be `stdout`

Reply:
This doesn't add up really. We expect `output` to be a filename, if we expect `stdout` as ivalue it is inconsistent. They don't have the same meaning or refer to the same thing. Arguably we shoul document the default case better.

Reply:
> We expect output to be a filename.

Sure, and if you so wished you should be able to pass `/dev/stdout` to it and it'd just work. `stdout` is in this case a symbolic name that resolves to that path in practice.

Reply:
Plus, it documents the behaviour, from the user perspective, it makes it explicit in the output of `vegeta encode -h` that the output will be to stdout, instead of that being implicit.